### ISSUE TITLE
No Color for Critical Log Level

### ISSUE THREAD

**Issue by ksolie** [metachris/logzero#120]
* logzero version: 1.5.0
* Python version: 2.7, 3.6
* Operating System: Windows 7

### Description
I Have a python application that is logging at the `CRITICAL` level and there is no color associated with that log level.

### What I Did
```
logger.critical(msg)
```

**Comment by sdhutchins** [comment]
See #12.

Also, my implementation is below.

```python
"""Main logging class to make logging easier."""
import os
import sys
from logzero import setup_logger, LogFormatter, logging, colors


class LogIt(object):
    """LogIt makes logging easier by creating easy loggers."""

    def __init__(self):
        """Initialize the logger format based on system platform."""
        # Set the different formats based on user's platform
        if sys.platform == 'win32':
            self._archive_format = '%m-%d-%Y_%I-%M-%p'
        elif sys.platform == 'linux':
            self._archive_format = '%m-%d-%Y@%I:%M:%S-%p'

        self._date_format = '%b-%d-%Y at %I:%M:%S %p'  # Used to add date
        self._log_format = ("%(color)s[%(levelname)s | %(name)s] [%(asctime)s | "
                            "%(module)s - line %(lineno)d]:%(end_color)s %(message)s")
        self._formatter = LogFormatter(fmt=self._log_format,
                                       datefmt=self._date_format)

        # Add a color for the critical level
        self._formatter.DEFAULT_COLORS[50] = colors.Fore.LIGHTRED_EX
        # Changed color for the debug level
        self._formatter.DEFAULT_COLORS[10] = colors.Fore.LIGHTBLUE_EX
        self.logging = logging

    def default(self, logname, logfile):
        """Create a log handler using default formatting.
        :param logname: Name of the log.
        :param logfile: Path to the logfile. Can be `None`.
        :return: A default_log object is returned.
        """

        default_log = setup_logger(name=logname.upper(), logfile=logfile,
                                   level=logging.DEBUG,
                                   formatter=self._formatter)

        assert isinstance(default_log, object)
        return default_log
```

**Comment by metachris** [comment]
Fixed in #339
